qt/python/mantidqt/widgets/tableworkspacedisplay/test_helpers/mock_qtable.py

```diff
@@ -0,0 +1,7 @@
+from mock import Mock
```

Comment:
Missing the licence header.

Can we have some module or class documentation to explain its purpose?